tests/Strategies/DynamicStrategyTest.php

```diff
@@ -0,0 +1,169 @@
+<?php
```

Comment:
I am just wondering if we should move this test into a `tests/Strategies/StrategyTest.php` as part of this PR as it means the Strategy testing all happens in the same area:

https://github.com/mikefrancis/laravel-unleash/blob/master/tests/UnleashTest.php#L255-L310